en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md

```diff
@@ -1,94 +1,121 @@
 # Create Your First REST API Proxy
 
-Choreo is a low-code cloud-native platform for building integration solutions. In this tutorial, you will do the following to explore REST API Proxies in Choreo:
+This tutorial introduces REST API proxies. 
+
+An API proxy fronts an API backend and intercepts the requests to the REST API. You can apply security, rate limiting, etc., to the REST API Proxy.
+
+In this tutorial, you will learn how to do the following:
 
 1. Expose an existing API by creating a REST API proxy
 2. Deploy the REST API proxy
 3. Test the REST API proxy to verify its functionality
-4. Manage your REST API proxy by adding rate limiting and security policies and leveraging the platform's API management capabilities.
+4. Manage your REST API proxy by adding rate limiting and security policies, thereby leveraging the platform's API management capabilities.
 
 !!!tip "Before you begin..."
     Familiarize yourself with REST API Proxies in Choreo. [REST API Proxies](../../develop/components/api-proxies/api-proxy.md) 
 
 
-Let's begin...
+Let's begin!
+
 ## Step 1: Create a REST API Proxy
 
 You can create a REST API proxy in one of the following methods: 
 
-- Design a new REST API proxy using an existing endpoint. 
-- Importing an OpenAPI specification of the existing API.
 
-In this tutorial, you will be importing an OpenAPI specification and creating a REST API proxy:
+In this tutorial, you will be providing a URL to a sample OpenAPI definition of a sample 
```

Comment:
This part is a little confusing: sample OpenAPI definition of a sample. Can we reword?

Reply:
Changed to "In this tutorial, you will be providing a URL to an OpenAPI definition of a sample API. "